backend/controllers/prisonerProfile/prisonerQuickLook.ts

```diff
@@ -46,6 +46,21 @@ const createFinanceLink = (offenderNo, path, value) =>
     formatCurrency(value || 0)
   }</a>`
 
+const getNextIepReviewDate = (iepSummary: { iepTime?: string }): string | undefined => {
+  const lastReviewDate = iepSummary?.iepTime
+  if (!lastReviewDate) return undefined
+  // TODO: nextReviewDate will come from incentivesApi in future
+  const nextReviewDate = moment(lastReviewDate, 'YYYY-MM-DD HH:mm').add(1, 'years')
```

Comment:
You could have a similar logic to try and get it from the API response - and if not present sum 1 year like you already do.

This would mean that when the API will start to have this field DPS will automatically use it (and then this logic could be removed at a later stage)

something like:

```JavaScript
let nextReviewDate = iepSummary?.nextReviewDate
if (!nextReviewDate) {
  // fallback while Incentives API doesn't return this field
  nextReviewDate = moment(lastReviewDate, 'YYYY-MM-DD HH:mm').add(1, 'years')
}
```

it's just an idea :)

Reply:
`let nextReviewDate = iepSummary?.nextReviewDate` that's exactly what's coming "soon". once the incentives api starts providing this (even if somewhat incorrectly at first)

Reply:
But you could have this in place right now and the UI would start displaying that `nextReviewDate` coming from the Incentives API as soon as this is in the response :)

Reply:
1. UI checks if this field is in the API response, if not calculate `nextReviewDate` as it currently does
2. Incentives API adds `nextReviewDate` to the IEP summary response (as in 1 year to begin with). UI starts to automatically display value from Incentives API
3. clean up UI code to remove fallback code (not needed anymore once this field is in Incentives API)

That was my thinking behind this idea.